### ISSUE TITLE
Visual updates to the team page

### ISSUE THREAD

**Issue by Marcuzz** [everyday-as/gmodstore-issues#850]
## Describe a feature you would like to have added to GmodStore
I think there could be made multiple improvements to the teams page. More specifically what I am referring to are these pages: https://www.gmodstore.com/teams/2887

**Backgrounds**

Allow team owner to select a background similar to script submissions.
This would allow teams to significantly make their team page stand out.

**Displaying members**

Instead of having a separate tab for "Members", show them on the first page the user sees.
![image](/uploads/5e72d66593d9c782f70ff31fbc4c1713/image.png)

Above is an example of how members could easily fit on the first page.
Why? I think it will give users who check out the team a higher chance of noticing it's member. Simply seeing the reputation of said developers could improve the changes of buyers purchasing the script.

Should just be a matter of taking the code for the members page and putting it where it shows "2 members" to display the members with their reputation? yaya ez pzz

**Show curated picked reviews**

Similar to where the "Buyer Rating" is shown, it might or might not be worth showing a few reviews IF they are curator featured on one of the teams scripts.

**Team avatars**

Basically this https://git.everyday.no/gmodstore/gmodstore-issues/issues/249
